core/services/ocr2/delegate.go

```diff
@@ -419,6 +397,49 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 	}
 }
 
+func getEVMTransmitterID(jb *job.Job, chainSet evm.ChainSet, chainID int64, lggr logger.SugaredLogger) (string, error) {
```

Comment:
But this is strictly EVM? It makes sense to me to rename it to getEVMEffectiveTransmitterID, because I imagine there could exist a` getCosmosTransmitterID`. Or do you mean that EffectiveTransmitter is an EVM only concept

Reply:
I agree on the renaming to be evm locked but make sure we add Effective to the signature